ViewGeneratorCore/tools/webpack.config.js

```diff
@@ -0,0 +1,69 @@
+﻿const glob = require('glob');
+const path = require('path');
+
+const bindingsMap = {};
+const entryMap = {};
+const outputMap = {};
+
+/** 
+ *  Get input and output entries from ts2lang file
+ * */
+let bindings = require(path.resolve('./ts2lang.json'));
+bindings.tasks.forEach(t => bindingsMap[t.input] = t.output);
+
+/** 
+ *  Build entry and output mappings for webpack config
+ * */
+Object.keys(bindingsMap).forEach(input => {
+    let output = bindingsMap[input];
+    glob.sync(input).forEach(f => {
+
+        // Exclude node_modules files
+        if (!f.includes("node_modules")) {
+
+            let entryName = path.parse(f).name;
+            entryMap[entryName] = ['./' + f];
+            outputMap[entryName] = output;
+        }
+    });
+});
+
+module.exports = {
+
+    target: 'node',
+
+    entry: entryMap,
+
+    externals: {
```

Comment:
I will use this to deal with these dependencies, and I will create a separate bundle containing React, React-Dom and ViewFrame in ReactViewResources